<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>568</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>570</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>800</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>800</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
font: 13pt &quot;Trebuchet MS&quot;;
</string>
    </property>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>50</y>
       <width>201</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(57, 57, 209);
color:rgb(255, 255, 255);
font: 18pt &quot;Trebuchet MS&quot;;
border-radius: 10px;</string>
     </property>
     <property name="text">
      <string>Editar Perfil</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_volver">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>31</width>
       <height>61</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(57, 57, 209);
border-radius: 10px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>Icon/back.png</normaloff>Icon/back.png</iconset>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>180</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Nombre (s)</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>180</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Apellido Paterno</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>240</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Domicilio</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>300</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Telefono</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="line_nombre">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>200</y>
       <width>201</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="maxLength">
      <number>30</number>
     </property>
    </widget>
    <widget class="QLineEdit" name="line_app">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>200</y>
       <width>191</width>
       <height>20</height>
      </rect>
     </property>
     <property name="maxLength">
      <number>50</number>
     </property>
    </widget>
    <widget class="QLineEdit" name="line_domicilio">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>260</y>
       <width>401</width>
       <height>20</height>
      </rect>
     </property>
     <property name="maxLength">
      <number>100</number>
     </property>
    </widget>
    <widget class="QLineEdit" name="line_telefono">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>320</y>
       <width>201</width>
       <height>20</height>
      </rect>
     </property>
     <property name="maxLength">
      <number>20</number>
     </property>
    </widget>
    <widget class="QPushButton" name="button_foto">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>30</y>
       <width>121</width>
       <height>121</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 60px;
border:1px solid black;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>Icon/user.png</normaloff>Icon/user.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>80</width>
       <height>80</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="button_guardar">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>460</y>
       <width>181</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(131, 131, 255);
color: rgb(255, 255, 255);
border-radius: 3px;</string>
     </property>
     <property name="text">
      <string>Guardar</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>Icon/save.png</normaloff>Icon/save.png</iconset>
     </property>
    </widget>
    <widget class="QFrame" name="frame">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>90</y>
       <width>711</width>
       <height>391</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid gray;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>150</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 1px solid white;</string>
      </property>
      <property name="text">
       <string>Edad</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="line_edad">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>170</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="maxLength">
       <number>10</number>
      </property>
     </widget>
     <widget class="QLineEdit" name="line_especialidad">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>230</y>
        <width>321</width>
        <height>21</height>
       </rect>
      </property>
      <property name="maxLength">
       <number>500</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>210</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 1px solid white;</string>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
      <property name="text">
       <string>Especialidad</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="line_usuario">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>330</y>
        <width>201</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="maxLength">
       <number>20</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>310</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Usuario</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>310</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Contrase√±a</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="line_password">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>330</y>
        <width>201</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="maxLength">
       <number>20</number>
      </property>
     </widget>
     <widget class="QLineEdit" name="line_password_2">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>330</y>
        <width>201</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="maxLength">
       <number>20</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>310</y>
        <width>161</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Repetir Contrase√±a</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>90</y>
        <width>151</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Apellido Materno</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="line_apm">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>110</y>
        <width>181</width>
        <height>20</height>
       </rect>
      </property>
      <property name="maxLength">
       <number>50</number>
      </property>
     </widget>
     <widget class="QLineEdit" name="line_correo">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>280</y>
        <width>411</width>
        <height>20</height>
       </rect>
      </property>
      <property name="maxLength">
       <number>20</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>260</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Correo</string>
      </property>
     </widget>
     <zorder>label_6</zorder>
     <zorder>line_edad</zorder>
     <zorder>line_especialidad</zorder>
     <zorder>label_4</zorder>
     <zorder>line_usuario</zorder>
     <zorder>label_7</zorder>
     <zorder>label_8</zorder>
     <zorder>line_password</zorder>
     <zorder>line_password_2</zorder>
     <zorder>label_9</zorder>
     <zorder>label_10</zorder>
     <zorder>line_apm</zorder>
     <zorder>line_correo</zorder>
     <zorder>label_11</zorder>
    </widget>
    <widget class="QPushButton" name="button_eliminar">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>460</y>
       <width>181</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(232, 119, 121);
color:rgb(255,255,255) ;
border-radius: 3px;</string>
     </property>
     <property name="text">
      <string>Eliminar</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>Icon/trash.png</normaloff>Icon/trash.png</iconset>
     </property>
    </widget>
    <zorder>frame</zorder>
    <zorder>pushButton</zorder>
    <zorder>button_volver</zorder>
    <zorder>label</zorder>
    <zorder>label_2</zorder>
    <zorder>label_3</zorder>
    <zorder>label_5</zorder>
    <zorder>line_nombre</zorder>
    <zorder>line_app</zorder>
    <zorder>line_domicilio</zorder>
    <zorder>line_telefono</zorder>
    <zorder>button_foto</zorder>
    <zorder>button_guardar</zorder>
    <zorder>button_eliminar</zorder>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
